Pokedex_ios/ViewModel/SortPokemonListViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  SortPokemonListViewModel.swift
+//  Pokedex_ios
+//
+//  Created by admin on 7/15/22.
+//
+
+import UIKit
+import Foundation
+
+class SortPokemonListViewModel {
+
+    var pokemons: [PokemonRaw]
+    weak var sortButtonActionDelegateDelegate: SortButtonActionDelegateDelegate?
+
+    init(pokemons: [PokemonRaw]) {
+        self.pokemons = pokemons
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Remove this init if its not needed or its not implemented

Reply:
Checked and indeed it was not necessary, removed already